test/asdf_elvish.bats

```diff
@@ -3,12 +3,15 @@
 load test_helpers
 
 setup() {
+  # shellcheck disable=SC1007
   export XDG_CONFIG_HOME= XDG_DATA_HOME= XDG_DATA_DIRS=
 
   local version=
   version=$(elvish -version)
 
+  # shellcheck disable=SC1007
   local ver_major= ver_minor= ver_patch=
+  # shellcheck disable=SC2034
```

Comment:
I changed everything except the `required_elvish_version_major` because I thought the meaning was already quite clear with `ver_major` and `ver_minor` comparison, next to the `skip` message.

Your changes also fix a bug in which the test's wouldn't properly show "skipped" if `elvish` wasn't installed. (my bad for that, I was the author that originally proposed these minimum version requirements).

There was also a bug in the "skip" message - the earliest version supported is 0.18 - the conditional was right, but the message was fixed to match.